app/main.js

```diff
@@ -692,6 +696,17 @@ function animate() {
     }
 }
 
+function resetBit() {
```

Comment:
why is it "resetBit" ?

Reply:
The tiles that make up the maze walls come from a "bit map". The initial render of the walls is `drawBit()` and this function resets the walls around pac-man when he dies. I guess a better name would be `resetTile()` or `resetWall()`. I'll open an issue to remind us to go back and change function and variable names to be more accurate.